tests/unit/validation/test_input_validation.py

```diff
@@ -277,7 +278,8 @@ def original_data() -> dict[ComponentType, np.ndarray]:
 
 @pytest.fixture
 def original_data_columnar(original_data):
-    return compatibility_convert_row_columnar_dataset(original_data, Ellipsis, DatasetType.input)
+    data_filter = ComponentAttributeFilterOptions.ALL
+    return compatibility_convert_row_columnar_dataset(original_data, data_filter, DatasetType.input)
```

Comment:
probably good to parametrize

Reply:
more specific?

Reply:
i mean to have this fixture  parametrized for both `ComponentAttributeFilterOptions.ALL` and `ComponentAttributeFilterOptions.RELEVANT`

Reply:
Best attempt for now pushed. Will further polish in the follow-up.